pallets/loans/rpc/src/lib.rs

```diff
@@ -0,0 +1,93 @@
+use std::sync::Arc;
+
+use codec::Codec;
+use jsonrpc_core::{Error as RpcError, ErrorCode, Result};
+use jsonrpc_derive::rpc;
+pub use pallet_loans_rpc_runtime_api::LoansApi as LoansRuntimeApi;
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::FixedU128;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+
+#[rpc]
+pub trait LoansApi<BlockHash, AccountId> {
+    #[rpc(name = "pallet_loans_get_account_liquidity")]
```

Comment:
This jsonrpc should be named `lonas_getAccountLiquidity`.

Reply:
@yz89 Please check latest code

Reply:
Latest code is `loans_get_account_liquidity` not `loans_getAccountLiquidity`

Reply:
Done